src/main/java/ai/elimu/util/audio/GoogleCloudTextToSpeechHelper.java

```diff
@@ -0,0 +1,98 @@
+package ai.elimu.util.audio;
+
+import ai.elimu.model.enums.Language;
+import com.google.cloud.texttospeech.v1beta1.AudioConfig;
+import com.google.cloud.texttospeech.v1beta1.AudioEncoding;
+import com.google.cloud.texttospeech.v1beta1.SsmlVoiceGender;
+import com.google.cloud.texttospeech.v1beta1.SynthesisInput;
+import com.google.cloud.texttospeech.v1beta1.SynthesizeSpeechResponse;
+import com.google.cloud.texttospeech.v1beta1.TextToSpeechClient;
+import com.google.cloud.texttospeech.v1beta1.VoiceSelectionParams;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import javax.transaction.NotSupportedException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Utility class for connecting to Google Cloud Text-to-Speech (https://cloud.google.com/text-to-speech/) 
+ * to synthesize an audio file from text.
+ * <p />
+ * 
+ * Supported voices and languages: https://cloud.google.com/text-to-speech/docs/voices
+ * <p />
+ * 
+ * Supported audio formats: https://cloud.google.com/speech-to-text/docs/encoding
+ * <p />
+ * 
+ * Google Cloud Console: https://console.cloud.google.com/apis/api/texttospeech.googleapis.com
+ * <p />
+ * 
+ * See sample code at https://github.com/googleapis/java-texttospeech/blob/master/samples/snippets/src/main/java/com/example/texttospeech/SynthesizeTextBeta.javas
+ */
+public class GoogleCloudTextToSpeechHelper {
+    
+    private static Logger logger = LogManager.getLogger();
+    
+    public static byte[] synthesizeText(String text, Language language) throws NotSupportedException, IOException {
+        logger.info("synthesizeText");
+        
+        byte[] byteArray = null;
+        
+        if ((language != Language.BEN)
+                && (language != Language.ENG)
+                && (language != Language.FIL)
+                && (language != Language.HIN)) {
+            throw new NotSupportedException("This language (" + language + ") is not yet supported: https://cloud.google.com/text-to-speech/docs/voices");
+        }
+        String languageCode = null;
```

Comment:
Is this supposed to be the [`ISO 639-1`](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code?